pydantic/json_schema.py

```diff
@@ -1172,13 +1243,50 @@ def schema(
     return json_schema
 
 
-def model_schema(
-    model: type[BaseModel],
+# TODO: The JSON schema cache has been there from the beginning (https://github.com/pydantic/pydantic/pull/190 —
```

Comment:
I think removing it is fine, might be worth moving this comment to the PR body, it'll both make more sense and arguably be more visible there.